goalgorilla/open_social
# Issue #2908231 by maikelkoopman: make display_submitted configurable

For all content types it should be configurable if the author and submitted date are displayed

How to test:
- Revert features (social_event, social_topic and social_book)
- Go to edit {content type} form and enable or disable Display author and date information under the tab Display settings.
- Check in the Node detail page the field is shown / hidden.

By default Event and Topic show the data. Book and Page will not show the data. Each installtion can now override this config via the UI.
